osu.Game/Overlays/Profile/ProfileHeader.cs

```diff
@@ -31,7 +31,9 @@ public ProfileHeader()
             User.ValueChanged += e => updateDisplay(e.NewValue);
 
             TabControl.AddItem(LayoutStrings.HeaderUsersShow);
-            TabControl.AddItem(LayoutStrings.HeaderUsersModding);
+
+            // todo: pending implementation.
+            // TabControl.AddItem(LayoutStrings.HeaderUsersModding);
```

Comment:
As I mentioned on discord, the whole control looks a bit weird without this. But I guess it's okay?

Reply:
Leaving it in place seems best until it's implemented.